krypto/src/tests/integration/test_hooks.py

```diff
@@ -19,7 +19,7 @@
 @pytest.fixture(scope='session')
 def definition_dir(krypto_kompile: Callable[..., Path]) -> Path:
     definition = """
-        requires "plugin/krypto.md"
```

Comment:
Can you add an extra directory to the Nix output, i.e. `src/plugin/krypto.md`? This way, this sort of hierarchical import work in a `kdist`-like setup.